drivers/serial/uart_ifx_cat1_pdl.c

```diff
@@ -34,6 +34,39 @@ LOG_MODULE_REGISTER(uart_ifx_cat1, CONFIG_UART_LOG_LEVEL);
 #define IFX_CAT1_UART_OVERSAMPLE_MAX              16UL
 #define IFX_CAT1_UART_MAX_BAUD_PERCENT_DIFFERENCE 10
 
+#ifdef CONFIG_UART_ASYNC_API
+#include <zephyr/drivers/dma.h>
+#include <cy_trigmux.h>
+
+struct ifx_cat1_dma_stream {
+	const struct device *dev_dma;
```

Comment:
```suggestion
	const struct device *dma_dev;
```
to be consistent with the below two variables

Reply:
corrected